lib/web/apiserver_test.go

```diff
@@ -2422,6 +2466,7 @@ func TestInstallDatabaseScriptGeneration(t *testing.T) {
 			},
 		})
 	require.NoError(t, err)
+	verifySecurityResponseHeaders(t, re.Headers())
```

Comment:
I considered something similar.  But it seemed too complicated to try and exercise the entire API space in a single test (we would basically need to go through the sum of all the actions being done here).  That's why I was instead trying to put verification around existing responses we conveniently already had.

We could create a single test and only test one or two api endpoints, but I was hoping to get more verification coverage here (particularly since I am trying to verify my centralized fixes are functioning broadly)

I also considered we could have the tests save all the results and then we can have a single test execute at the end and review / verify them.  But this would introduce test ordering requirements :/

All in all, I don't really like how this testing looks, and I know it's not complete.  Some non-ideal solution will probably need to be accepted, so let me know you think might be best.

Reply:
I was able to detect and fix the issue mentioned yesterday by spreading this testing pattern further through this test class.  @zmb3 I would appreciate your perspective on if I should:
1) Leave this check spread throughout the test file
2) Keep this pattern but minimize it to just a handful of points
3) Adopt one of the strategies above

I personally lean towards option 1 because it's most complete with minimal code, but I am happy with any solution.